prime-router/src/main/kotlin/fhirengine/translation/hl7/config/TruncationConfig.kt

```diff
@@ -5,7 +5,7 @@ package gov.cdc.prime.router.fhirengine.translation.hl7.config
  */
 data class TruncationConfig(
     val truncateHDNamespaceIds: Boolean,
-    val truncateHl7Fields: List<String>,
+    val truncateHl7Fields: List<String> = emptyList(),
```

Comment:
Why did this need a default?  It compiled for me even after removing it

Reply:
it hit issue at jackson object mapping at I guess...

Reply:
Seems like something is off then... if the field was non-nullable before I'm not sure why it would need a default now.

Reply:
in legacy code, a list field always populated with [], some the corresponding constructor does not default..., but having [] value every where for those fields not having a value yet produce many fields with [] which is actually "not present"